src/site/content/en/blog/install-thumbor/index.md

```diff
@@ -0,0 +1,184 @@
+```
+
+```bash
+# computer vision packages
+sudo apt-get install -y python-opencv libopencv-dev
+```
+
+```bash
+# image format packages
+sudo apt-get install -y libjpeg-dev libpng-dev libwebp-dev webp
+```
+
+## Install Thumbor
+
+Install Thumbor using pip.
+
+```bash
+sudo pip install thumbor
+```
+
+Note: Many Python developers use [virtualenv](https://pypi.org/project/virtualenv/) to manage their packages. For the sake of simplicity, these instructions do not use virtualenv. If you are installing Thumbor in a standalone environment, virtualenv is not necessary. If you choose to use virtualenv, note that Thumbor requires Python 2.7 and will not work with newer versions of pip (e.g. these instructions use pip 8.1.1).
+
+If you’ve successfully installed Thumbor, this should work:
+
+```bash
+thumbor --help
+```
+
+## Run Thumbor
+
+Run Thumbor. Debug logging is optional but can be helpful when you’re getting started.
+
+```bash
+thumbor --log-level debug
+```
+
+Thumbor is now running.
+
+
+## Open Firewall Port
+
+By default, Thumbor runs on port 8888. If your VM’s IP address is `12.123.12.122`, then you would access Thumbor from the web browser at `http://12.123.12.123:8888/.../$IMAGE`.
+
+However, this probably won’t work for you (yet) because cloud providers usually require that you explicitly open firewall ports before they will accept incoming traffic.
+
+Update the firewall to expose port 8888. Here’s more information on how to do this for: [Google Cloud](https://cloud.google.com/vpc/docs/using-firewalls), [AWS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html), and [Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/nsg-quickstart-portal). Note that for Google Cloud you need to first [assign a static IP address to your VM](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-external-ip-address) and then [allow an external HTTP connection](https://cloud.google.com/vpc/docs/special-configurations#externalhttpconnection).
+
+## Try It Out
+
+Thumbor is now accessible and ready for use. Try it out by visiting the following URL:
+
+```
+http://YOUR_VIRTUAL_MACHINE:8888/unsafe/100x100/https://web.dev/install-thumbor/hero.jpg
+```
+
+Note that this URL uses HTTP. Thumbor uses HTTP by default but can be [configured](https://thumbor.readthedocs.io/en/latest/image_loader.html) to use HTTPS.
+
+
+You should see an image that is 100 pixels wide by 100 pixels tall. Thumbor has taken the image `hero.jpg` and size specified in the URL string and served the result. You can replace the image in the URL string (i.e `https://web.dev/install-thumbor/hero.jpg`) with any other image (e.g. `https://your-site.com/cat.jpg`) and Thumbor will resize that image too. 
+
+
+## Use Thumbor
+
+You’ve successfully installed Thumbor. Next, learn about using Thumbor in the [Use Thumbor to Optimize Images](https://web.dev/codelab-thumbor) codelab.
```

Comment:
Use sentence case for the title. `Use Thumbor to Optimize Images` => `Use Thumbor to optimize images`

https://web.dev/handbook/grammar/#titles-and-headings